sds/src/scoped_ruleset/mod.rs

```diff
@@ -217,7 +217,7 @@ where
         // Sanitize the segment and push it. If the segment is an Index, it will push None.
         self.sanitized_segments_until_node.push(segment.sanitize());
 
-        let true_positive_rules_count = if self.should_keywords_match_event_paths {
+        let true_positive_rules_count = if false {
```

Comment:
I will fix this in a later PR, because I would like first to verify it actually increases the performances on staging

Reply:
Not as a full review, We don't want to merge this PR as it is because it will disable the included keyword on path feature. Instead of merging it, I think you can manually bump the version in the shared library with this PR and then create an "experimental" image directly from the other PR.

Reply:
It doesn't disable the feature, we don't perform the check at the same place is all it is doing. 

I put `if false` here so that I don't have to remove all the code before confirming that we're improving the performances

Reply:
oh, I see, in that case I think it's better to comment the code + add a quick explanation  at the beginning. Once merged, without looking at the git history, the code would look like wrong 🙏

Reply:
Yes I agree. In my mind, I was going to write a PR right after to cleanup the code after making sure it's indeed performing better. But agreed, a comment is better for future readers of the code 👍

Reply:
[Add comments regarding the manual disablement of the feature](https://github.com/DataDog/dd-sensitive-data-scanner/pull/144/commits/9fdf11f984bdc9d9d8da2eb1a49ea7ff6293d4dc)